code/modules/jobs/job_types/engineering.dm

```diff
@@ -77,7 +77,7 @@ Station Engineer
 	access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
```

Comment:
we are on a server with vore, if someone can eat another person, they can grind up souless bodies and eat them

engineers repair the structures, scientists maintain the machinery